### PR TITLE
Use best practices outlined in #286.

### PR DESCRIPTION
**Pull request by kolodny**

### CODE REVIEW

File: node/hello-world/index.js [hunk 1]
```diff
@@ -39,18 +46,27 @@ const getRowGreeting = (row) => {
       key: `greeting${index}`,
       data: {
         [COLUMN_FAMILY_NAME]: {
-          [COLUMN_NAME]: greeting,
+          [COLUMN_NAME]: {
+            value: greeting,
+            timestamp: new Date(),
```
Comment (sduskis) [inline node/hello-world/index.js]: Please add a comment saying that setting the date allows the client to perform retries.  If server-side time is used, retries may cause multiple cells to be generated.

File: node/hello-world/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// A version of this script is available for node v4 in index.v4.js
+// A version of this script is available for node v6 in index.v6.js
```
Comment (sduskis) [inline node/hello-world/index.js]: Can you please add a comment here for the node.js version for which this sample will work?

**Comment by sduskis** [comment]
@kolodny, can we close this PR, or do we still need this?

**Comment by kolodny** [comment]
This PR can be closed.

**Pull request closed without merging**